src/settings/groups/AddGroupDialog.ts

```diff
@@ -37,28 +37,37 @@ export class AddGroupDialog implements Component<AddGroupDialogAttrs> {
 	view(vnode: Vnode<AddGroupDialogAttrs>): Children {
 		const { availableGroupTypes, groupType, availableDomains, onEmailChanged, onBusyStateChanged } = vnode.attrs
 		return [
-			m(DropDownSelector, {
-				label: "groupType_label",
-				items: availableGroupTypes.map((t) => {
-					return {
-						name: getGroupTypeDisplayName(t),
-						value: t,
-					}
-				}),
-				selectedValue: groupType,
-				selectionChangedHandler: vnode.attrs.onGroupTypeChanged,
-			}),
+			availableGroupTypes.length > 1
```

Comment:
we should probably at some point redesign this dialog that it either gets all the data it needs for each group type or that it only ever expects the data for one type